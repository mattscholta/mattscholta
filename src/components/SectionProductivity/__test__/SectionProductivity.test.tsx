import * as React from 'react';
import { render, RenderResult } from '@testing-library/react';

import {
  SectionProductivity,
  SectionProductivityProps
} from '../SectionProductivity';

describe('SectionProductivity', () => {
  let props: SectionProductivityProps;
  let wrapper: RenderResult;

  const createWrapper = () => <SectionProductivity {...props} />;

  beforeEach(() => {
    props = {};
    wrapper = render(createWrapper());
  });

  it('matches snapshot', () => {
    expect(wrapper.container.childNodes).toMatchSnapshot();
  });
});
