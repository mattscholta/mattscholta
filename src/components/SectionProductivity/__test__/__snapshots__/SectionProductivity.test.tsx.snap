// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionProductivity matches snapshot 1`] = `
NodeList [
  <section>
    <h2
      class="ui-heading"
    >
      Productivity
    </h2>
    <span
      class="ui-subhead"
    />
    <p>
      These apps are just useful for navigating the day-to-day. Some of which I am highly satisfied with like Rocket, 1Pass and BetterTouch, and the others are just necessary.
    </p>
    <ul
      style="column-count: 3; text-align: center;"
    >
      <li
        class="u-py-2x"
      >
        <a
          class="u-text-24"
          href="https://1password.com"
          rel="noopener noreferrer"
          target="_blank"
          title="🔐 1Password"
        >
          🔐 1Password
        </a>
      </li>
      <li
        class="u-py-2x"
      >
        <a
          class="u-text-24"
          href="https://folivora.ai"
          rel="noopener noreferrer"
          target="_blank"
          title="👆 BetterTouch"
        >
          👆 BetterTouch
        </a>
      </li>
      <li
        class="u-py-2x"
      >
        <a
          class="u-text-24"
          href="https://matthewpalmer.net/rocket"
          rel="noopener noreferrer"
          target="_blank"
          title="🚀 Rocket App"
        >
          🚀 Rocket App
        </a>
      </li>
      <li
        class="u-py-2x"
      >
        <a
          class="u-text-24"
          href="https://slack.com"
          rel="noopener noreferrer"
          target="_blank"
          title="💬 Slack App"
        >
          💬 Slack App
        </a>
      </li>
      <li
        class="u-py-2x"
      >
        <a
          class="u-text-24"
          href="https://sparkmailapp.com"
          rel="noopener noreferrer"
          target="_blank"
          title="📧 Spark Email"
        >
          📧 Spark Email
        </a>
      </li>
      <li
        class="u-py-2x"
      >
        <a
          class="u-text-24"
          href="https://apps.apple.com/us/app/tweetdeck-by-twitter/id485812721"
          rel="noopener noreferrer"
          target="_blank"
          title="🐦 TweetDeck"
        >
          🐦 TweetDeck
        </a>
      </li>
    </ul>
  </section>,
]
`;
